code/languages/Java/Handlingexceptions/README.md

```diff
@@ -0,0 +1,7 @@
+#Handling exceptions in java
```

Comment:
@Vaidehi0421 it should be `# Handling exceptions in java`.Also, why to add `Handling exceptions in java` heading two times?

Reply:
@iamrajiv No need to add the link?

Reply:
@Vaidehi0421 I think there is no problem if the document has a link in the heading or you can make like this i.e. adding the link to the bottom of document something like this

Article [Exceptions in Java](https://www.geeksforgeeks.org/exceptions-in-java/) discusses the different types of exceptions and how to deal with them.

Reply:
@Vaidehi0421 I think there is no problem if the document has a link in the heading or you can make like this i.e. adding the link to the bottom of document something like this
Article [Exceptions in Java](https://www.geeksforgeeks.org/exceptions-in-java/) discusses the different types of exceptions and how to deal with them.